datadog_checks_downloader/setup.py

```diff
@@ -60,8 +60,6 @@ def parse_pyproject_array(name):
         'Intended Audience :: Developers',
         'Topic :: System :: Monitoring',
         'License :: OSI Approved :: BSD License',
-        'Programming Language :: Python :: 2',
```

Comment:
Since Python 2 is no longer supported for the downloader, we can delete `setup.py` and any other files that `pyproject.toml` is supposed to replace (see [here](https://github.com/DataDog/integrations-core/pull/11433) for Snowflake example)

Reply:
I've looked into it and simply removing `setup.py` would work and be enough. Submitted separately as #13508 for ease of review.